plugins/search/src/components/SearchType/SearchType.Accordion.test.tsx

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import user from '@testing-library/user-event';
+
+import { SearchContext } from '../SearchContext';
+import { SearchType } from './SearchType';
+
+describe('SearchType.Accordion', () => {
+  const contextSpy = {
+    result: { loading: false, value: { results: [] } },
+    term: '',
+    types: [],
+    filters: {},
+    toggleModal: jest.fn(),
+    setTerm: jest.fn(),
+    setTypes: jest.fn(),
+    setFilters: jest.fn(),
+    setPageCursor: jest.fn(),
+  };
+
+  const expectedLabel = 'Expected Label';
+  const expectedType = {
+    value: 'expected-type',
+    name: 'Expected Type',
+    icon: <></>,
+  };
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('should render as expected', () => {
+    const { getByText } = render(
+      <SearchContext.Provider value={contextSpy}>
```

Comment:
any specific reason to why you decided to use SearchContext.Provider here instead of SearchContextProvider which has an initial value already, which means you don't need to mock it here?

Reply:
Good call!  I think I had other assertions in mind when I started writing this and then they turned out not to be relevant.  Can update.